pkg/controllers/infra-services-controller.go

```diff
@@ -70,11 +70,13 @@ func applyDefaultRBACs(ctx context.Context, cfg *config.Config) error {
 			"controllers/kube-controller-manager/csr_approver_clusterrole.yaml",
 			"controllers/cluster-policy-controller/namespace-security-allocation-controller-clusterrole.yaml",
 			"controllers/cluster-policy-controller/podsecurity-admission-label-syncer-controller-clusterrole.yaml",
+			"controllers/cluster-policy-controller/podsecurity-admission-label-privileged-namespaces-syncer-controller-clusterrole.yaml",
```

Comment:
As we discussed on slack, let's move these RBAC changes to cluster-policy-controller.go so they're closer to the code they affect.

Reply:
Nevermind, these are cluster roles and we want them all managed in one place.